server/models/questions/removeEditor.js

```diff
@@ -0,0 +1,34 @@
+const { pool } = require('../database')
+
+/**
+ *
+ * @param {*} param0
+ * @param {String} param0.username
+ * @param {String} param0.editor
+ * @param {Number} param0.questionId
+ * @return {Promise}
+ */
+
+function removeEditor({ username, editor, questionId }) {
+  return new Promise((resolve, reject) => {
+    pool.query(
+      `DELETE FROM questions_editors a WHERE
+       EXISTS(SELECT 1 FROM (SELECT * FROM questions_editors) b WHERE editor=? AND question_id=?)
+       AND editor=?`,
```

Comment:
Try to avoid `SELECT *` as much as possible, when not needed, as it unnecessarily slows the query down.
The query can be simplified as:
```suggestion
      `DELETE FROM questions_editors a WHERE
       EXISTS(SELECT 1 FROM questions_editors WHERE editor=? AND question_id=?)
       AND editor=? AND question_id=?`,
```

And one more thing, we should ensure that creator of the question is not deleted from editors. So, that should also be done.

Reply:
This has to be done..cuz Mysql does not allow us to refer the table we are about to delete or update ..so we HAVE to make a copy of it..

Reply:
For the case where editor is a creator, I will take care of it

Reply:
Okay